sw/device/exts/common/meson.build

```diff
@@ -5,6 +5,11 @@
 # RISC-V linker parameters.
 riscv_linker_script = '@0@/@1@'.format(meson.source_root(), files(['flash_link.ld'])[0])
 riscv_linker_args = [
+  # The linker script includes an autogenerated definition of the available
+  # memory regions. The include path is given relative to the source root, so we
+  # need to add the source root to the linker lookup path (which is the same for
+  # libraries and linker scripts.
+  '-Wl,-L,@0@'.format(meson.source_root()),
```

Comment:
Is there a reason we can't make this a global argument, like we do with gcc's `-I` flags?